app/src/main/java/com/example/playlistmaker/ui/mediateka/playlists/NewPlaylistFragment.kt

```diff
@@ -0,0 +1,114 @@
+package com.example.playlistmaker.ui.mediateka.playlists
+
+import android.net.Uri
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.activity.OnBackPressedCallback
+import androidx.activity.result.PickVisualMediaRequest
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.core.content.ContextCompat
+import androidx.core.widget.doOnTextChanged
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.navigation.fragment.findNavController
+import com.example.playlistmaker.R
+import com.example.playlistmaker.databinding.FragmentCreatePlaylistBinding
+import com.example.playlistmaker.domain.models.search.Playlist
+import com.example.playlistmaker.ui.MainActivity
+import org.koin.androidx.viewmodel.ext.android.viewModel
+
+class NewPlaylistFragment : Fragment() {
+    private var _binding: FragmentCreatePlaylistBinding? = null
+    private val binding get() = _binding!!
+    private var coverUriSelect: Uri? = null
+
+    private val vm by viewModel<NewPlaylistViewModel>()
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        _binding = FragmentCreatePlaylistBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        (activity as MainActivity).hideNavBar()
```

Comment:
```suggestion
        (activity as? MainActivity)?.hideNavBar()
```
Safe cast предпочтительнее обычного каста почти во всех случаях